.github/workflows/release.yml

```diff
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - master
+    tags:
```

Comment:
We need to check whether the chart version was bumped when release on tagging.

Reply:
Already fixed by add steps `check_version`